plugin/blocks/flexwall-break/block.js

```diff
@@ -0,0 +1,22 @@
+( function ( blocks, element, blockEditor ) {
+    var el = element.createElement;
+
+    blocks.registerBlockType( 'flexwall/flexwall-break', {
```

Comment:
```suggestion
    blocks.registerBlockType( 'flexwall/break', {
```

@philipreichert Why don't you just call it `flexwall/break`? Seems to me like double namespacing.